_pages/camelot/project.md

```diff
@@ -0,0 +1,24 @@
+---
+permalink: /camelot/project
+layout: bunkhouse
+---
+
+The new bunkhouse will be located behind the existing cabin, about 100 meters up the hill. This is close enough that underground electrical wiring will be cost-effective, but far enough that sound from the cabin and campfire won’t carry. The bunkhouse will be farther from the property line than the existing cabin.
+
+<img src="/images/bunkhouse/bunkhouse-site-plan.png" alt="Bunkhouse site plan" width="400"/>
+
+The bunkhouse will have five sleeping rooms which accommodate 10 people each. A large central hallway contains ample space for gear storage for 50 people. The floor plan also includes a covered deck area.
+
+<img src="/images/bunkhouse/bunkhouse-floor-plan.png" alt="Bunkhouse floor plan" width="400"/>
```

Comment:
@dlaw you were just missing the "/" at the start of the path to indicate it's relative to the root of the website, not to the local path.